elixir/apps/web/lib/web/live/settings/identity_providers/new.ex

```diff
@@ -120,17 +122,24 @@ defmodule Web.Settings.IdentityProviders.New do
           type="radio"
           name="next"
           value={next_step_path(@adapter, @account)}
-          class={~w[ w-4 h-4 border-neutral-300 ]}
+          class={[
+            "w-4 h-4 border-neutral-300",
+            @opts[:enabled] == false && "cursor-not-allowed"
+          ]}
+          disabled={@opts[:enabled] == false}
           required
         />
         <.provider_icon adapter={@adapter} class="w-8 h-8 ml-4" />
         <label for={"idp-option-#{@adapter}"} class="block ml-2 text-lg text-neutral-900">
           <%= @name %>
         </label>
-        <%= if @enterprise_feature do %>
-          <.badge class="ml-2" type="primary" title="Feature available on the Enterprise plan">
-            ENTERPRISE
-          </.badge>
+
+        <%= if @opts[:enabled] == false do %>
+          <.link navigate={~p"/#{@account}/settings/billing"} class="ml-2 text-sm text-primary-500">
+            <.badge class="ml-2" type="primary" title="Feature available on a higher pricing plan">
+              UPGRADE TO UNLOCK
```

Comment:
I like the idea of giving the user the ability to quickly get to their billing account to upgrade and unlock a given feature, but in this case the only way to unlock this would be to have an enterprise account correct?  Which means they wouldn't be able to take immediate action on unlocking this feature.   I was wondering if maybe we should have this do something like display a message on how to upgrade to "Enterprise"?

Curious on everyones thoughts: @jamilbk @AndrewDryga

Reply:
This link already goes to the billing page, I've added "Contact Sales Team" there as a call to action and explained that some features are only available on Enterprise plan (the ones that are WIP or the ones we are not sure about). I think this is good enough for now for them to take action and request an upgrade, at least.
 
<img width="1451" alt="Screenshot 2024-02-20 at 13 55 05" src="https://github.com/firezone/firezone/assets/1877644/76b52424-14ed-4acd-b019-3fc857cfd562">

Reply:
I think all of that is good.  I definitely like the "Contact Sales Team" button for sure.  The one small concern I have is that in the screenshot you've posted, "Automatically sync users and groups" does not have a `(beta)` label on it, which means users may get confused and think they are able to get that feature without upgrading to an enterprise plan.  I'm not suggesting we mark the IDP sync as 'beta', just wondering if we should call out enterprise features differently.

Reply:
Maybe it will be better this way, so that entire section is just the list of reasons to upgrade.
<img width="1426" alt="Screenshot 2024-02-20 at 14 42 27" src="https://github.com/firezone/firezone/assets/1877644/76c1a2e7-21f3-4dc2-aaeb-49f71ace77c1">

Reply:
Hmm it may be better to just remove `(beta)` terminology throughout. It's a little tougher to sell beta features.